drivers/misc/ethos_u/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+# Copyright (c) 2022 Arm Limited.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+zephyr_sources(init.c)
```

Comment:
Did you consider using zephyr_library() and zephyr_library_sources() instead? It's not clear to me why this needs to use zephyr_sources(), and that's quite unusual for a driver: compare the output of running `git grep zephyr_sources drivers` with `git grep zephyr_library_sources` in the zephyr top level directory for examples.

Reply:
No particular reason for using `zephyr_sources`. If a Zephyr library is recommended I'll use that.